src/murmur/UnixMurmur.cpp

```diff
@@ -340,7 +340,11 @@ void UnixMurmur::finalcap() {
 	if (getrlimit(RLIMIT_RTPRIO, &r) != 0) {
 		qCritical("Failed to get priority limits.");
 	} else {
-		qWarning("Resource limits were %ld %ld", r.rlim_cur, r.rlim_max);
+		static_assert(sizeof(long long unsigned int) >= sizeof(rlim_t), "rlim_t is unexpectedly large");
```

Comment:
https://wiki.sei.cmu.edu/confluence/display/c/INT35-C.+Use+correct+integer+precisions

Reply:
I am wondering whether
```cpp
std::numeric_limits<unsigned long long int>::max() >= std::numeric_limits<rlim_t>::max()
```
would also solve the issue without causing UB since this is comparing 2 integers of (potentially) different type (and size) with each other :thinking: 
Do you happen to know the answer to that?

Reply:
Okay I asked on SO (https://stackoverflow.com/q/66991556/3907364) and I was told that this is not UB, since it falls in the application of so-called "integer promotions".